tests/integrate_test/test_integrate_existent_scores.py

```diff
@@ -123,8 +123,7 @@ def test_existent_score(self):
 
         prev_block, tx_results = self._make_and_req_block([tx1])
         self._write_precommit_state(prev_block)
-        self.assertTrue("is a directory. Check " in tx_results[0].failure.message)
-        self.assertEqual(tx_results[0].status, int(False))
+        self.assertEqual(tx_results[0].status, int(True))
```

Comment:
that logic don't need to check revision code.
but come to think of it, I'm afraid that it is good to edit that test result.

In my opinion, that test files don't need to test due to fix logic that you fixed.
so I ask you remove you check them in advance and remove them.